repo/packages/B/bookkeeper/0/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "assets": {
+    "container": {
+      "docker": {
+        "bookkeeper": "zhaijia/bookie:latest"
```

Comment:
lets not use `latest`. Package might break with future change. Use a fixed version.